slapr/__main__.py

```diff
@@ -19,11 +19,12 @@
     ),
     slack_channel_id=os.environ["SLACK_CHANNEL_ID"],
     slapr_bot_user_id=os.environ["SLAPR_BOT_USER_ID"],
+    number_of_approvals_required=max(1, int(os.environ.get("SLAPR_NUMBER_OF_APPROVALS_REQUIRED", 1))),
     emoji_review_started=os.environ.get("SLAPR_EMOJI_REVIEW_STARTED", "review_started"),
-    emoji_approved=os.environ.get("SLAPR_EMOJI_APPROVED", "tests-passed"),
+    emoji_approved=os.environ.get("SLAPR_EMOJI_APPROVED", "approved"),
     emoji_needs_change=os.environ.get("SLAPR_EMOJI_CHANGES_REQUESTED", "changes_requested"),
-    emoji_merged=os.environ.get("SLAPR_EMOJI_MERGED", "merged-purple"),
-    emoji_closed=os.environ.get("SLAPR_EMOJI_CLOSED", "rip"),
+    emoji_merged=os.environ.get("SLAPR_EMOJI_MERGED", "merged"),
+    emoji_closed=os.environ.get("SLAPR_EMOJI_CLOSED", "closed"),
```

Comment:
Our Banno repos are using custom emojis for these 3. In all the Banno workflows I've seen the env vars match what's here so reverting these 3 lines should be safe.

I plan to upstream these changes back into the DataDog/slapr repo so we don't have to maintain our own fork